hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -700,7 +732,11 @@
             sources.
 
             This verifies ALL wake up sources. This also verifies that the pwrmgr sequencing is
-            working correctly as expected. X-ref'ed with all individual IP tests.
+            working correctly as expected. X-ref'ed with all individual IP tests. For each wakeup
+            source clear and enable `wake_info` CSR, bring the chip to low power, optionally disable
+            the source's clock, disable all wakeup sources, issue a wakeup, verify the pwrmgr
+            `wake_status` CSR is off, enable the wakeup source, verify `wake_info` indicates the
```

Comment:
IMO checking if the `wake_status` CSR is working correctly is an IP level test. After low power entry, the CPU is not available, so we cannot enter low power with all wake sources disabled anyway. We can simplify this.

Reply:
I see your point... it is unclear what this test can do in addition to the IP test. Perhaps all we need here is a connectivity check for the wakeups? I simplified the test description anyway.

Reply:
IMO this whole part can be handled by the IP level - at chip level it should be sufficient to ensure that `wake_info` is reflecting the correct source. 

```
            optionally disable
            the source's clock, disable all wakeup sources, issue a wakeup, verify the pwrmgr
            `wake_status` CSR is off,
```